Makefile

```diff
@@ -31,6 +33,8 @@ pi:
 stop:
 	docker stop $(docker ps -a -q --filter "name=$(name)") && docker rm $(docker ps -a -q --filter "name=$(name)")
 
+
```

Comment:
Did you mean to add these blank lines?

Reply:
No, forgotten artifacts from playing with test commands. Will clean up.